From cf49402689758b89bb8342cd4903268924e7b97a Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Thu, 30 Apr 2020 13:47:08 +0300
Subject: [PATCH] [SW-12005] hl: boot-fit: add loadaddr property

The loadaddr property will be used to program the boot image descriptors
with the address in RAM the image should be loaded to before further
work with it - either execution or any other work on the image.

Change-Id: I69b281c3b66e613fc501cc8a5054d933148a7a40
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/board/habanalabs/common/uboot.its b/board/habanalabs/common/uboot.its
index 0cc12c6..ff02f05 100644
--- a/board/habanalabs/common/uboot.its
+++ b/board/habanalabs/common/uboot.its
@@ -80,6 +80,7 @@
 			arch = "arm64";
 			compression = "none";
 			offset = <0x000a0000>, <0x008a0000>;
+			loadaddr = <0x7F 0xF0004000>;
 			hash {
 				algo = "sha256";
 			};
@@ -92,6 +93,7 @@
 			arch = "arm64";
 			compression = "none";
 			offset = <0x000b0000>, <0x008b0000>;
+			loadaddr = <0x7F 0xF0100000>;
 			hash {
 				algo = "sha256";
 			};
